main.py

```diff
@@ -819,11 +819,11 @@ def format_severity(severity: int) -> str:
 
 
 def format_diagnostic(diagnostic: Diagnostic) -> str:
-    (line, character) = diagnostic.range.start
-    location = "{}:{}".format(line + 1, character + 1)
-    formattedMessage = diagnostic.message.replace("\n", "").replace("\r", "")
-    return "\t{:<8}\t{:<8}\t{:<8}\t{}".format(
```

Comment:
About this leading tab, does the result_file_regex still work now that you have removed it? 
If it's not too much effort, a screenshot showing the visual changes would be appreciated. I've been trying to keep up with checking out your PRs locally, but it would more productive to just read the PR and approve :)